atl-checker/src/game_structure/lcgs/ir/symbol_checker.rs

```diff
@@ -277,6 +278,7 @@ impl<'a> SymbolChecker<'a> {
     }
     /// First combines all numbers, as we already know the max of that
     /// Then returns a new checked Vec of Expr to find Max of.
+    #[allow(clippy::unnecessary_unwrap)]
```

Comment:
```suggestion
    #[allow(clippy::unnecessary_wraps)]
```

Reply:
I have tried` #[allow(clippy::unnecessary_wraps)]` but i am getting this warning.

![image](https://user-images.githubusercontent.com/38072340/116655085-bdcf6a00-a98a-11eb-846e-10c3dc232edd.png)

Reply:
Huh. But `unwrap` doesn't remove the clippy warning either. I guess, just skip this one then.

Reply:
After i updated my clippy, it seems like it recognize `clippy:unnecessary_wraps` and the warning doesn't appear for me anymore.